packages/fxa-shared/db/models/auth/security-event.ts

```diff
@@ -58,8 +59,10 @@ export class SecurityEvent extends BaseAuthModel {
   }) {
     const id = uuidTransformer.to(uid);
     const ipAddrHmac = ipHmac(Buffer.from(ipHmacKey), id, ipAddr);
+    console.log('you got into the create action.');
+
     try {
-      await SecurityEvent.callProcedure(
+      await this.callProcedure(
```

Comment:
Can we make this change across all functions in this class? I am sure it'll catch someone else off guard at some point if we don't.

Reply:
Looks like there is still one class refrence to SecurityEvent.query(...) in the findByUidAndIP method.

Reply:
Oh interesting... I think for consistency sake it makes sense to switch it to this.